coral/src/app/features/configuration/users/Users.tsx

```diff
@@ -0,0 +1,85 @@
+import { TableLayout } from "src/app/features/components/layouts/TableLayout";
+import { useQuery } from "@tanstack/react-query";
+import { getUserList } from "src/domain/user";
+import { UsersTable } from "src/app/features/configuration/users/components/UsersTable";
+import { Pagination } from "src/app/components/Pagination";
+import { useSearchParams } from "react-router-dom";
+import {
+  useFiltersContext,
+  withFiltersContext,
+} from "src/app/features/components/filters/useFiltersContext";
+import TeamFilter from "src/app/features/components/filters/TeamFilter";
+import { SearchFilter } from "src/app/features/components/filters/SearchFilter";
+
+function UsersWithoutFilterContext() {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const { teamName, search } = useFiltersContext();
+
+  const currentPage = searchParams.get("page")
+    ? Number(searchParams.get("page"))
+    : 1;
+
+  const {
+    data: users,
+    isLoading,
+    isError,
+    error,
+  } = useQuery(["get-user-list", currentPage, teamName, search], {
```

Comment:
Nitpick: I think we didn't decide it formally, but I think we mostly use "endpoint name" as a key for queries? So, `getUserList` in this case.

Reply:
You're absolutely right. 🤦  @ myself

Reply:
https://github.com/Aiven-Open/klaw/pull/1971/commits/d30aef80a59d1a5c92decf698fa6e38a3f8439eb